sematic/resolvers/resource_managers/resource_manager.py

```diff
@@ -0,0 +1,33 @@
+# Standard Library
+import abc
+from typing import List
+
+# Sematic
+from sematic.external_resource import ExternalResource
+
+
+class ResourceManager(abc.ABC):
```

Comment:
`AbstractResourceManager` to align with the rest of the code base?
Also move it one level up?